src/rss_etl.py

```diff
@@ -0,0 +1,175 @@
+from supabase import create_client, Client
+import feedparser
+import os
+import re
+from enum import Enum
+from dataclasses import dataclass
+
+"""
+File for extracting data from the RSS feeds of the PA Senate and House of Representatives
+and loading it into Supabase.
+
+The database tables updated by this script are:
+    Bills
+    Revisions
+"""
+
+# from dotenv import load_dotenv
+# load_dotenv()  # for local secret management with .env file
+
+senate_rss_feed = "https://www.legis.state.pa.us/WU01/LI/RSS/SenateBills.xml"
+house_rss_feed = "https://www.legis.state.pa.us/WU01/LI/RSS/HouseBills.xml"
+
+# sb_api_url = os.getenv("SUPABASE_API_URL")  # local secret management with .env file
+# sb_api_key = os.getenv("SUPABASE_API_KEY")  # local secret management with .env file
+sb_api_url = os.environ["SUPABASE_API_URL"]  # github actions secret management
+sb_api_key = os.environ["SUPABASE_API_KEY"]  # github actions secret management
```

Comment:
Don't need both versions of this code --if you have one, you don't need the other.